docs/genesis-preparation.md

```diff
@@ -0,0 +1,352 @@
+            "vssKey": "WCECtpe8B/5XPefEhgg7X5veUIYH/RRcvXbz6w7MIJBwWYU="
+        },
+        ...
+    }
+}
+```
+
+#### 5.2.1. Format Explanation
+
+Section `"avvmDistr"` contains AVVM keys with corresponding coins values.
+
+Section `"blockVersionData"` contains fundamental blockchain-related values:
+
+*  `"heavyDelThd"` - heavyweight delegation threshold,
+*  `"maxBlockSize"` - maximum size of block, in bytes,
+*  `"maxHeaderSize"` - maximum size of block's header, in bytes,
+*  `"maxProposalSize"` - maximum size of Cardano SL update proposal, in bytes,
+*  `"maxTxSize"` - maximum size of transaction, in bytes,
+*  `"mpcThd"` - threshold for participation in MPC,
+*  `"scriptVersion"` - script version, 
+*  `"slotDuration"` - slot duration, in microseconds,
+*  `"softforkRule"` - rules for softfork:
+   *  `"initThd"` - initial threshold, right after proposal is confirmed,
+   *  `"minThd"` - minimal threshold (i.e. threshold can't become less than this one),
+   *  `"thdDecrement"` - theshold will be decreased by this value after each epoch,
+*  `"txFeePolicy"` - transaction fee policy's values,
+*  `"unlockStakeEpoch"` - unlock stake epoch,
+*  `"updateImplicit"` - update implicit period, in slots,
+*  `"updateProposalThd"` - threshold for Cardano SL update proposal,
+*  `"updateVoteThd"` - threshold for voting for Cardano SL update proposal.
+
+Please note that values of all thresholds are fractional numbers here. So if particular threshold is
+defined as `X`, it's actual value is `X * 10^-15`.
+
+Section `"bootStakeholders"` contains Bootstrap stakeholders' identifiers (see step `1.2`) with
+corresponding weight.
+
+Field `"ftsSeed"` contains seed value required for Follow-the-Satoshi mechanism.
+
+Section `"protocolConsts"` contains basic protocol constants:
+
+*  `"k"` - security parameter from the paper,
+*  `"protocolMagic"` - protocol magic value, 
+*  `"vssMaxTTL"` - VSS certificates maximum timeout to live (number of epochs),
+*  `"vssMinTTL"` - VSS certificates minimum timeout to live (number of epochs).
+
+Section `"heavyDelegation"` contains an information about heavyweight delegation:
+
+*  `"cert"` - delegation certificate,
+*  `"delegatePk"` - delegate's public key,
+*  `"issuerPk"` - stakeholder's (issuer's) public key,
+*  `"omega"` - index of epoch the block PSK is announced in, it is needed for replay attack prevention.
+
+Field `"startTime"` contains timestamp for the proper start of the cluster.
+IOHK DevOps team should provide proper start time.
+
+Section "vssCerts" contains VSS certificates:
+
+*  `"expiryEpoch"` - index of epoch when this certificate will expire,
+*  `"signature"` - signature of certificate,
+*  `"signingKey"` - key used for signing,
+*  `"vssKey"` - VSS public key (can be read from node's secret key, as shown in step `1.2`).
+
+**Responsible**: IOHK DevOps team and someone from Cardano SL Core team.
+
+### 5.3. Valdate Genesis Data
+
+Created genesis JSON should be validated.
```

Comment:
How/when the JSON file is used?

Given that multiple filenames were mentioned the mechanism for specifying which is used should be documented.

Reply:
@jmitchell this file is to be shipped with cardano (to end users, to cluster)

Reply:
Filename probably something like `genesis-mainnet.json`